proposals/lip-0063.md

```diff
@@ -565,25 +565,25 @@ Once the snapshot block is final, a patched version of Lisk Core v4 will be rele
 
 The URL can be configured. By default, the URL points to a server of the Lisk Foundation. The snapshot block of this hard fork is validated by checking that its block ID matches the hard coded block ID in the patched version of Lisk Core v4\. The snapshot block of the previous migration is validated by checking that its block ID matches with the `previousBlockID` property of its child block (see also [here](#getting-the-lisk-core-3-block-history)). Hence, there are no security concerns by downloading them from even a non-trusted server. Users can nevertheless configure their node to download it from a server of their choice. The snapshot blocks could be provided by any user. This may also be helpful in situations where the Lisk Foundation server is down.
 
+### Assigning BLS Key to Validator Accounts
+
+In blockchains created with Lisk SDK v6, one needs to [register](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0044.md#registervalidatorkeys) a [valid BLS key](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0038.md#public-key-registration-and-proof-of-possession) in order to become a validator. This results in an exception for Lisk Core v4 as the existing validators do not have a valid BLS key right after the hard fork. Since the DPoS module expects that each validator account has a BLS key and a proof of possession set in the snapshot block (see [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#genesis-state-initialization)), this will be done by assigning a fixed public key for which signature validation will always fail along with a dummy value for the proof of possession. This proof of possession will never be evaluated as the DPoS and the validator module handle an exception for this particular case (see [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#genesis-state-finalization) and [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0044.md#registervalidatorwithoutblskey)). Every validator must then register a valid BLS key via a _[register keys](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0050.md#register-keys)_ transaction in order to become an active validator, ideally within the [bootstrap period](#bootstrap-period) described below.
```

Comment:
Should `Validator` here begin with a capital letter, since we're referring to the module?
```
This proof of possession will never be evaluated as the DPoS and the validator module...
```

```suggestion
In blockchains created with Lisk SDK v6, one needs to [register](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0044.md#registervalidatorkeys) a [valid BLS key](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0038.md#public-key-registration-and-proof-of-possession) in order to become a validator. This results in an exception for Lisk Core v4 as the existing validators do not have a valid BLS key right after the hard fork. Since the DPoS module expects that each validator account has a BLS key and a proof of possession set in the snapshot block (see [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#genesis-state-initialization)), this will be done by assigning a fixed public key for which signature validation will always fail along with a dummy value for the proof of possession. This proof of possession will never be evaluated as the DPoS and the Validator module handle an exception for this particular case (see [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#genesis-state-finalization) and [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0044.md#registervalidatorwithoutblskey)). Every validator must then register a valid BLS key via a _[register keys](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0050.md#register-keys)_ transaction in order to become an active validator, ideally within the [bootstrap period](#bootstrap-period) described below.
```